stable/mongodb/templates/_helpers.tpl

```diff
@@ -86,3 +86,15 @@ Return the proper image name (for the metrics image)
 {{- printf "%s/%s:%s" $registryName $repositoryName $tag -}}
 {{- end -}}
 
+
+{{ template "mongodb.initdbScriptsCM" . }}
```

Comment:
This template should be used in the statefulset definition, otherwise it would not work